### ISSUE TITLE
Article Page (FRONT)

### ISSUE THREAD

**Issue by thejoefriel** [yalla-coop/working-class-history#21]
__Related issue(s)__

_Add any issues here that may be dependent or must be completed before this issue can be tackled_

__Notes__

_Add any useful notes here_

---

### Acceptance Criteria:

_REMEMBER THAT WHOEVER WORKS ON THIS ISSUE MUST TICK OFF ALL THE POINTS IN THIS LIST UNLESS THERE IS CLEAR AGREEMENT IN THE COMMENTS TO SAY OTHERWISE. **DO NOT REVIEW A PR INVOLVING THIS ISSUE UNLESS THIS HAS BEEN DONE**_ 

- [ ] Url is /article/:id/:articleName
- [ ] Create page https://www.figma.com/file/j6eCLlivlp6hiOusYQYpVY/Wireframes?node-id=33%3A1777
- [ ] Image is optional so make sure it's designed to work without image there 
- [ ] Map embed will be covered in separate issue so for now leave out
- [ ] All content would come from Baserow (to be connected in another issue) 
- [ ] Feed through the year to the timeline component #17
